7.Lifecycle.md

```diff
@@ -0,0 +1,29 @@
+**Jam3 Lessons - React**
+
+# Lifecycle
+
+> Each component has several "lifecycle methods" that you can override to run code at particular times in the process.
+Methods prefixed with will are called right before something happens, and methods prefixed with did are called right after something happens.
```

Comment:
I think it sounds better with this changes:

`Each component has several _methods_ that you can override to run code at particular moments of a component's _life_.`

For the second paragraph I would delete it and list all the lifecycle callbacks briefly explaining when they are executed so it actually serves a quick reference for any developer.

Let me know what you think @neo.